account_analytic_nrq/views/account_analytic_line_views.xml

```diff
@@ -1,32 +1,27 @@
-<?xml version="1.0" encoding="utf-8"?>
+<?xml version="1.0" encoding="utf-8" ?>
 <odoo>
-
-    <record id="view_account_analytic_line_tree" model="ir.ui.view">
-        <field name="name">account.analytic.line.tree</field>
-        <field name="model">account.analytic.line</field>
-        <field name="inherit_id"
-               ref="analytic.view_account_analytic_line_tree"/>
-        <field name="arch" type="xml">
-            <xpath expr="//field[@name='account_id']" position="after">
-                <field name="product_id"/>
-                <field name="partner_id"/>
-                <field name="move_partner_id"/>
-                <field name="user_id"/>
-            </xpath>
-        </field>
-    </record>
-
-    <record id="view_account_analytic_line_filter" model="ir.ui.view">
-        <field name="name">account.analytic.line.select</field>
-        <field name="model">account.analytic.line</field>
-        <field name="inherit_id"
-               ref="analytic.view_account_analytic_line_filter"/>
-        <field name="arch" type="xml">
-            <xpath expr="//field[@name='account_id']" position="after">
-                <field name="move_partner_id"/>
-                <field name="user_id"/>
-            </xpath>
-        </field>
-    </record>
-
+  <record id="view_account_analytic_line_tree" model="ir.ui.view">
+    <field name="name">account.analytic.line.tree</field>
+    <field name="model">account.analytic.line</field>
+    <field name="inherit_id" ref="analytic.view_account_analytic_line_tree" />
+    <field name="arch" type="xml">
+      <xpath expr="//field[@name='account_id']" position="after">
+        <field name="product_id" />
+        <field name="partner_id" />
+        <field name="move_partner_id" />
+        <field name="user_id" />
+      </xpath>
+    </field>
+  </record>
+  <record id="view_account_analytic_line_filter" model="ir.ui.view">
+    <field name="name">account.analytic.line.select</field>
+    <field name="model">account.analytic.line</field>
+    <field name="inherit_id" ref="analytic.view_account_analytic_line_filter" />
+    <field name="arch" type="xml">
+      <xpath expr="//field[@name='account_id']" position="after">
+        <field name="move_partner_id" />
+        <field name="user_id" />
+      </xpath>
+    </field>
+  </record>
```

Comment:
@TimLai125 I guess the indentation should not be like this (should be 4 instead of 2).  It currently don't have .editorconfig in this repo. Is it how it's supposed to be?  I see this https://github.com/qrtl/rpl-private/blob/2e559bb2470f09355087a73958b46ddfc0ac0fe2/.editorconfig#L5 in rpl-private.